CSharp/Samples/PizzaBot/PizzaOrderDialog.cs

```diff
@@ -108,7 +108,7 @@ enum Days { Saturday, Sunday, Monday, Tuesday, Wednesday, Thursday, Friday };
 
             var days = (IEnumerable<Days>)Enum.GetValues(typeof(Days));
             PromptDialog.Choice(context, StoreHoursResult, days, "Which day of the week?",
-                descriptions: from day in days

```

Comment:
>descriptions [](start = 16, length = 12)

Introduction of breaking changes to primitive dialogs of SDK, e.g. PromptDialogs, should be minimized!